### ISSUE TITLE
No video in project

### ISSUE THREAD

**Issue by gongalov71** [OpenShot/openshot-qt#5155]
**Question:**
When starting a new project in OpenShot project files are green and video is not playing, only fastened audio. Doesn't matter what is the video format, mp4 or mkv or the file size, problem is the same. How can I fix this?

**System Details:**
- OpenShot Version 3.0.0: {{ log_context.openshot_qt_version }}
- Operating System / Distro: Windows 11

Screenshots:

![openshot 2023-02-23 210409](https://user-images.githubusercontent.com/115870458/221006987-d1f9ac1b-a1b6-4308-aa95-6383050f8523.png)
![openshot project file](https://user-images.githubusercontent.com/115870458/221007795-545d5881-a560-405a-a429-06509688fc78.png)
![openshot timeline](https://user-images.githubusercontent.com/115870458/221008113-286fcd57-cf87-45af-bd85-cc9171a4afb0.png)

**Comment by Colorjet3** [comment]
Hello @gongalov71.  Please do this:

1.  close openshot.
2.  go to openshot.org/download/#daily and download the latest DEV Daily build of v3.0.0.
3.  Install it.
4.  Start Openshot.
5.  Go to Edit | Preferences | Preview tab and ensure that the Default Audio Sample Rate is set to 48000.  If you had to change it to this value then ensure to restart Openshot.
6.  Go to Edit | Preferences | Performance tab.
- "Hardware Decoder Mode" should be set to **No Acceleration**
- "Hardware Decoder Graphics Card" should be set to "Graphics Card 1"
- "Hardware Encoder Graphics Card" should be set to "Graphics Card 1".

**Note:**  Hardware acceleration is very experimental and very limited in Windows environment.

7.  Restart Openshot and give it a try.

If you are still having issues do this:
1.  Close Openshot.
2.  Backup c:\users\username\.openshot_qt folder.  The entire folder.
3.  Delete c:\users\username\.openshot_qt folder.
4.  Start Openshot and give it a try.

Let me know the result and if needed we can continue to troubleshoot.

**Comment by gongalov71** [comment]
I installed the daily build, then deleted the folder and video has been fixed but audio is still fast and not editable. Just for the test I tried to create a short video from two others and after rendering in the output file audio is normal. I just checked the settings again, and the Default Audio Sample Rate was at 22050. I'm sure I set it to 48000, but it didn't saved. Nope, it just doesn't accept 48000, after restart it's set back to 22050.
I think I've fixed it. I just played with Playback Audio Device. First it was set to Windows Audio: Speakers (SoundBlasterX AE-5 Plus) and the Default Audio Sample Rate was at 22050. I changed the Playback Audio Device to DirectSound: Speakers (SoundBlasterX AE-5 Plus) and after restart the Default Audio Sample Rate was automatically set to 48000

![openshot settings](https://user-images.githubusercontent.com/115870458/221051648-fa31169d-3d17-4081-ae3d-c96c53e64998.png)
